### PR TITLE
portsyncd: Bringing up VLAN members and removing members logic

### PR DESCRIPTION
**Pull request by stcheng**

### CODE REVIEW

File: portsyncd/linksync.cpp [hunk 1]
```diff
@@ -91,16 +109,30 @@ void LinkSync::onMsg(int nlmsg_type, struct nl_object *obj)
...
+        string member_key = m_ifindexNameMap[master] + ":" + key;
 
-        if (nlmsg_type == RTM_DELLINK)
-            m_vlanTableProducer.del(key);
+        if (nlmsg_type == RTM_DELLINK) /* Will it happen? */
+            m_vlanTableProducer.del(member_key);
         else /* RTM_NEWLINK */
         {
             FieldValueTuple t("tagging_mode", "untagged");
             fvVector.push_back(t);
```
Comment (qiluo-msft) [inline portsyncd/linksync.cpp]: You can move fvVector to local block, and remove .clear(). Later you can define another fvVector, not to reuse the old one.

Comment (stcheng) [inline portsyncd/linksync.cpp]: resolved

Comment (qiluo-msft) [inline portsyncd/linksync.cpp]: closed.

**Comment by qiluo-msft** [comment]
No issue from me. Please also check with other reviewers.

**Pull request merged**